File: course-p-oop/POOUber/Java/UberPool.java
package Java;

public class UberPool extends Car{
    String brand;
    String model;

    public UberPool(String license, Account driver, String brand, String model){
        super(license, driver); //super refer to objects in Car
        this.brand = brand; // this refer to loca objects
        this.model = model;
    }
}

File: course-p-oop/POOUber/JS/UberPool.js
//Native JS
// UberPool.prototype = new Car(license, driver, brand, model)
// {
//     this.brand = brand;
//     this.model = model;
// }

//EcmaScript 6
class UberPool extends Car{
    constructor(license, driver, brand, model){
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }
}